### ISSUE TITLE
Make SECRET_KEY an environment Config Var and remove from settings.py

### ISSUE THREAD

**Issue by erwa** [cardinalitypuzzles/smallboard#13]
Currently, our Django SECRET_KEY is checked in to smallboard/settings.py. This key may possibly be used for salting, generating hashes and tokens, and signing data, according to https://stackoverflow.com/questions/7382149/purpose-of-django-setting-secret-key. We should move this out of settings.py and make it an environment Config Var (https://dashboard.heroku.com/apps/smallboard/settings) instead. (We should also change the key since the current one has been checked in.)

**Comment by erwa** [comment]
Fixed in aa921d0de9d5bd5bfecfc49ef3876eb306152d59.

**Comment by erwa** [comment]
Added a `DJANGO_SECRET_KEY` environment variable. If not set, a random secret key is generated using:
```
from django.core.management.utils import get_random_secret_key

get_random_secret_key()
```

We had accidentally checked in our original SECRET_KEY, so I generated a new one and set it as the `DJANGO_SECRET_KEY` environment variable at https://dashboard.heroku.com/apps/smallboard/settings.
